bluemira/base/parameter_frame/_frame.py

```diff
@@ -115,14 +186,133 @@ def to_dict(self) -> Dict[str, Dict[str, Any]]:
     @classmethod
     def _validate_parameter_field(cls, field: str) -> Tuple[Type, ...]:
         member_type = cls.__dataclass_fields__[field].type
-        if (member_type is not NewParameter) and (
+        if (member_type is not Parameter) and (
             not hasattr(member_type, "__origin__")
-            or member_type.__origin__ is not NewParameter
+            or member_type.__origin__ is not Parameter
         ):
-            raise TypeError(f"Field '{field}' does not have type NewParameter.")
+            raise TypeError(f"Field '{field}' does not have type Parameter.")
         value_types = get_args(member_type)
         return value_types
 
+    @classmethod
```

Comment:
Does this one need to be a `classmethod`?

Reply:
it could be directly a function now with some work I've just done but I set it up as a classmethod so ti can be called after `validate_parameter_field` in the `from_dict` classmethod